include/vcpkg/commands.zbootstrap-standalone.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <vcpkg/commands.interface.h>
+
+namespace vcpkg::Commands
+{
+    struct BootstrapStandaloneCommand : BasicCommand
```

Comment:
Hmmm OK, for Z things I think putting them in the command makes sense since we never expect to remove the Z, for X things I think they should not be born with X.